servers.json

```diff
@@ -1,3 +1,11 @@
+[ 
```

Comment:
This is not properly formatted and you have not included your server logo in the logos directory.